packages/lms/src/lib/components/header/Settings.svelte

```diff
@@ -47,7 +47,7 @@
 
 <style>
 	.wrapper {
-		grid-area: settings;
+		/* grid-area: settings; not doing anything*/
```

Comment:
If something doesn't do anything, it's probably best to just remove it?

Reply:
That was a comment for myself that I forgot to remove. Done it now.